.github/workflows/performance-profiling.yml

```diff
@@ -3,7 +3,7 @@ name: ZenML CLI Performance Profiling
 on:
   pull_request:
     types: [opened, synchronize, ready_for_review]
-    paths: [src/zenml]
+    paths: [src/zenml/*, pyproject.toml]
```

Comment:
this won't cut it, you need to cover all of the subpaths (i.e. `src/zenml/**`) according to https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#patterns-to-match-branches-and-tags

Reply:
Fixed it. 😄